fedora-builder/src/test/java/org/dataconservancy/nihms/builder/fs/FilesystemModelBuilderTest.java

```diff
@@ -92,6 +93,10 @@ public void testElementValues() {
 
         DepositMetadata.Manuscript manuscriptMetadata = submission.getMetadata().getManuscriptMetadata();
         assertEquals("http://dx.doi.org/10.1039/c7fo01251a", manuscriptMetadata.getManuscriptUrl().toString());
+
+        //test that the DepositFileLabelMaker is being called to supply the file labels
+        String label = submission.getManifest().getFiles().get(0).getLabel();
+        assertTrue(label.endsWith(" type")); //both file descriptions in the json file are required to have labels, end with this
```

Comment:
@jrmartino just trying to understand wha tis going on here:

In `DepositFileLabelMaketTest` none of the labels end with the string ` type`, but in production the would?